requirements.txt

```diff
@@ -1 +0,0 @@
-nengo
```

Comment:
this pull request is quite old, nengo is already installed by setup so this was redundant, in fact i can just remove the requirements file